commands/clients/create.js

```diff
@@ -15,9 +15,15 @@ function * run (context, heroku) {
       redirect_uri: url
     }
   })
-  let client = yield cli.action(`Creating ${context.args.name}`, request)
-  cli.log(`HEROKU_OAUTH_ID=${client.id}`)
-  cli.log(`HEROKU_OAUTH_SECRET=${client.secret}`)
+  if (context.flags.shell) {
+    let client = yield request
+    cli.log(`HEROKU_OAUTH_ID=${client.id}`)
+    cli.log(`HEROKU_OAUTH_SECRET=${client.secret}`)
+  } else {
+    let client = yield cli.action(`Creating ${context.args.name}`, request)
```

Comment:
it looks like this is the only line that's different. Can those `cli.log()` lines be moved outside of the conditional?